sip/execution_control/configuration_db/redis/db_client/master_client.py

```diff
@@ -1,116 +1,214 @@
 # -*- coding: utf-8 -*-
 """High Level Master Controller Client API."""
-import ast
+import logging
 
-from .config_db_redis import ConfigDb
+from datetime import datetime
+from db_client import ConfigDb
```

Comment:
Is there a reason why this was changed to an absolute import? (I think it might be clearer to use relative imports here because the object `ConfigDb` is part of this package.)